openid-4-verifiable-presentations-1_0.md

```diff
@@ -1909,7 +1973,6 @@ The following is a non-normative example of a request that combines this specifi
 ```
 GET /authorize?
   response_type=vp_token%20id_token
-  &scope=openid
```

Comment:
This `scope=openid` should actually stay, as this section is showing an example of a SIOP request which does require `openid`. (openid should still be omitted in the new examples you're adding)

Reply:
Thanks! i wasn't sure about this earlier.
I've made the change.